test/ScrollBehavior.test.js

```diff
@@ -46,7 +48,15 @@ describe('ScrollBehavior', () => {
             (location) => {
               expect(location.state).to.not.exist();
               expect(scrollTop(window)).to.equal(15000);
-              history.push('/detail');
+              history.push('/detail#child2');
+            },
+            () => {
+              expect(child2().scrollTop).to.equal(0);
```

Comment:
is this the right check? this seems to be asserting on the scroll position _within_ this element, not how far the top of the window is from the element